bundles/specmate-integration-test/src/com/specmate/test/integration/ModelGenerationTestBase.java

```diff
@@ -31,19 +31,39 @@
 	private Predicate<JSONObject> MATCHES_VAR_COND(String var, String cond, String type) {
 		return (obj -> obj.getString(ECLASS).equals(CEGNode.class.getSimpleName())
 				&& obj.getString(RequirementsPackage.Literals.CEG_NODE__TYPE.getName()).equals(type)
-				&& (obj.getString(RequirementsPackage.Literals.CEG_NODE__VARIABLE.getName())).toLowerCase()
-						.equals(var.toLowerCase())
-				&& (obj.getString(RequirementsPackage.Literals.CEG_NODE__CONDITION.getName())).toLowerCase()
-						.equals(cond.toLowerCase()));
+				&& checkStringEquality(obj.getString(RequirementsPackage.Literals.CEG_NODE__VARIABLE.getName()), var)
+				&& checkStringEquality(obj.getString(RequirementsPackage.Literals.CEG_NODE__CONDITION.getName()), cond));
 	}
 
 	private Predicate<JSONObject> MATCHES_ID_VAR_COND(String id, String var, String cond) {
 		return (obj -> obj.getString(ECLASS).equals(CEGNode.class.getSimpleName())
 				&& obj.getString(BasePackage.Literals.IID__ID.getName()).equals(id)
-				&& (obj.getString(RequirementsPackage.Literals.CEG_NODE__VARIABLE.getName())).toLowerCase()
-						.equals(var.toLowerCase())
-				&& (obj.getString(RequirementsPackage.Literals.CEG_NODE__CONDITION.getName())).toLowerCase()
-						.equals(cond.toLowerCase()));
+				&& checkStringEquality(obj.getString(RequirementsPackage.Literals.CEG_NODE__VARIABLE.getName()), var)
+				&& checkStringEquality(obj.getString(RequirementsPackage.Literals.CEG_NODE__CONDITION.getName()), cond));
+	}
+	
+	
+	/**
+	 * Check if two strings contain the same words. Order and case are not relevant
+	 * @param s1
+	 * @param s2
+	 * @return
+	 */
+	private boolean checkStringEquality(String s1, String s2) {

```

Comment:
This method seems overly complex due to the many splits and replaces of differnt kinds of whitspace. How about:
1) split strings with regex e.g. using \s* (any whitespace, any number of times)
2) Add all non empty parts into a set
3) Compare sets using equals